Czaple/Attractions/Screens/AttractionView.swift

```diff
@@ -142,7 +148,7 @@ class AttractionView: UIViewController {
             player?.stop()
             geoButton.setTitle(NSLocalizedString("Przejdź do tablic", comment: ""), for: .normal)
         default:
-            AttractionManager.instance.boardAlert(controller: self)
+            boardAlert.boardAlertAction(controller: self)
```

Comment:
If you would let `AttractionView` to present this alert it would look like

```swift
let alert = boardAlert.boardAlertAction()
navigationController?.present(alert, animated: true)
```
you can see what is happening clearly